src/components/Search/SearchView.tsx

```diff
@@ -22,8 +22,15 @@ function SearchView(props: SearchViewProps) {
     useFocusInput();
 
   const navigate = useNavigate();
-  const goToCompareResultsPage = () => {
-    navigate('/compare-results', { replace: false });
+
+  const goToCompareResultsPage = (selectedRevisions: Revision[]) => {
+    const revisions = [...selectedRevisions.map(rev => rev.revision)];
+    const repos = [...selectedRevisions.map(rev => rev.repository_id)];
+    navigate({
+      pathname: '/compare-results', 
+      // how to get short form of revision id
```

Comment:
@kimberlythegeek  were you able to figure out how to get short form of the revisions

Reply:
as I understand it, the short hash is just the first 12 characters. you can use `truncateHash` from the helper function file for this